.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
+manifest:
+  version: 1.0
+
+
+triggers:
+  exclude:
+    branch:
+      - l10n_dev
```

Comment:
@coderabbitai Provide a complete list of all merged PRs for milestone 1.19.0, search for bug label then do another search for enhancement label, ensuring that all results are captured by handling pagination properly. Categorise the result into features and bug fixes, and put features first. Format the result as numbered list, and show PR number with the title.

In a separate sentence, give me the assignees usernames excluding jjw24 and sorted in case insensitive alphabetical order.

Reply:
There are more results. Ensure all results are captured by handling pagination properly.

Sort the assignees usernames case insensitive and alphabetically.

Reply:
There should be 52 bug fix results

Reply:
Why were you able to get it this time?